Python/bytecodes.c

```diff
@@ -0,0 +1,3983 @@
+#include "Python.h"
+
+#include "opcode.h"
+#include "pycore_atomic.h"
+#include "pycore_frame.h"
+
+void _PyFloat_ExactDealloc(PyObject *);
+void _PyUnicode_ExactDealloc(PyObject *);
+
+#define SET_TOP(v)        (stack_pointer[-1] = (v))
+#define GETLOCAL(i)     (frame->localsplus[i])
+
+#define inst(name, stack_effect) case name:
+#define family(name) static int family_##name
```

Comment:
There's also lots of red in the `family` section. The `family` macro expands to `static int family_xxx = ...`. The C preprocessor expands all of the following opcode names into integers, so what C sees is `static int family_xxx = 123, 456, 789, ...`. It complains that `456` and `789` aren't identifiers, since the commas make it think you're trying to declare a bunch of variables. So either the macro or the DSL may need updating here.

Reply:
Ah, that was a last-minute change, I had a `+` separator there first. I'll put it inside curlies and make the macro define an unsized array.